modules/custom/social_language/src/Plugin/Block/LanguageSwitcherBlock.php

```diff
@@ -22,6 +24,27 @@
  */
 class LanguageSwitcherBlock extends LanguageBlock {
 
+  /**
```

Comment:
We may want to move these new methods below the `build` method. I think the `build` method will be most interesting to people visiting this file.

Reply:
Shouldn't the blockAccess always be above the build?